test/integrations/destinations/criteo_audience/dataDelivery/business.ts

```diff
@@ -0,0 +1,352 @@
+import { generateProxyV1Payload } from '../../../testUtils';
+export const headers = {
+  Authorization: 'Bearer success_access_token',
+  'Content-Type': 'application/json',
+  Accept: 'application/json',
+};
+export const params = {
+  destination: 'criteo_audience',
+};
+const method = 'PATCH';
+
+const output = {
+  response: {
+    status: 200,
+    body: {
+      output: {
+        status: 200,
+        message: 'Request Processed Successfully',
+        response: [
+          {
+            error: '""',
+            metadata: {
+              attemptNum: 1,
+              destinationId: 'dummyDestinationId',
+              dontBatch: false,
+              secret: {},
+              sourceId: 'dummySourceId',
+              userId: 'dummyUserId',
+              workspaceId: 'dummyWorkspaceId',
+            },
```

Comment:
Metadata can be kept in a variable and be referred in the different place currently is getting duplicated in multiple places alternatively you can use this [func](https://github.com/rudderlabs/rudder-transformer/blob/9e047747a119cd3e23cb0c352363788f40e0ef42/test/integrations/testUtils.ts#L516)